### ISSUE TITLE
`mamba env update` failing at very end with certifi SSL issue

### ISSUE THREAD

**Issue by arjitj2** [mamba-org/mamba#1925]
```
done
#
# To activate this environment, use
#
#     $ conda activate root
#
# To deactivate an active environment, use
#
#     $ conda deactivate

Retrieving notices: ...working... failed
Traceback (most recent call last):
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 1125, in __call__
  File "/opt/mambaforge/lib/python3.10/site-packages/conda_env/cli/main.py", line 80, in do_call
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/core.py", line 75, in wrapper
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/core.py", line 39, in display_notices
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/http.py", line 36, in get_notice_responses
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/http.py", line 39, in <genexpr>
  File "/opt/mambaforge/lib/python3.10/concurrent/futures/_base.py", line 621, in result_iterator
  File "/opt/mambaforge/lib/python3.10/concurrent/futures/_base.py", line 319, in _result_or_cancel
  File "/opt/mambaforge/lib/python3.10/concurrent/futures/_base.py", line 458, in result
  File "/opt/mambaforge/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
  File "/opt/mambaforge/lib/python3.10/concurrent/futures/thread.py", line 58, in run
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/http.py", line 42, in <lambda>
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/cache.py", line 37, in wrapper
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/notices/http.py", line 58, in get_channel_notice_response
  File "/opt/mambaforge/lib/python3.10/site-packages/requests/sessions.py", line 600, in get
  File "/opt/mambaforge/lib/python3.10/site-packages/requests/sessions.py", line 587, in request
  File "/opt/mambaforge/lib/python3.10/site-packages/requests/sessions.py", line 701, in send
  File "/opt/mambaforge/lib/python3.10/site-packages/requests/adapters.py", line 460, in send
  File "/opt/mambaforge/lib/python3.10/site-packages/requests/adapters.py", line 263, in cert_verify
OSError: Could not find a suitable TLS CA certificate bundle, invalid path: /opt/mambaforge/lib/python3.10/site-packages/certifi/cacert.pem

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mambaforge/bin/mamba", line 11, in <module>
    sys.exit(main())
  File "/opt/mambaforge/lib/python3.10/site-packages/mamba/mamba.py", line 923, in main
  File "/opt/mambaforge/lib/python3.10/site-packages/mamba/mamba_env.py", line 196, in main
  File "/opt/mambaforge/lib/python3.10/site-packages/conda_env/cli/main.py", line 91, in main
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 1413, in conda_exception_handler
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 1128, in __call__
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 1170, in handle_exception
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 1181, in handle_unexpected_exception
  File "/opt/mambaforge/lib/python3.10/site-packages/conda/exceptions.py", line 1251, in print_unexpected_error_report
ModuleNotFoundError: No module named 'conda.cli.main_info'
The command '/bin/sh -c mamba env update -n root -f ${PKG_HOME}/environment.yml' returned a non-zero code: 1
make: *** [Makefile:18: build] Error 1
```

Part of our `environment.yml` is installing `python 3.8.5`; I'm suspecting that maybe the `cacert.pem` that requests is looking for is in the `3.8` dir rather than `3.10`.

**Comment by achimgaedke** [comment]
Same problem here...

**Comment by wolfv** [comment]
Thanks for letting us know - I think there is more than one issue at play :)

I think the certifi package is alright (I just checked). 

However, it looks like requests is instantiated after you down/upgrade python (if I understand correctly?). And there is a new feature in conda to look for "channel notifications". I will ping the conda team to see if they understand what's exactly happening.

I suspect that downgrading `conda` could solve this particular issue for the time being, but let me see with the conda team.

**Comment by wolfv** [comment]
It would be interesting to see if `CONDA_NUMBER_CHANNEL_NOTICES=0` helps to work around the issue in the meantime ... There are apparently some other issues with this new feature as well (https://github.com/conda/conda/issues/11516)

**Comment by travishathaway** [comment]
Hi all, (conda core-dev here 👋)

I just wanted to chime in because I was able to successfully recreate this issue consistently in `conda`. I'm also going to dump everything I know so far about this issue, including how to recreate it.

**TLDR:** This is not a `mamba` bug. This issue is occurring in `conda` because the base environment's Python is being updated in place (I inferred this was the case because of the presence of this line `#     $ conda activate root` in the initial error report), and the `notices` retrieval happens after this update has happened when the `cacert.pem` has been removed from where the current process knows to look. Metaphorically speaking, it's getting the rug pulled out from underneath it 😅.

### Recreating this bug:

Using this Dockerfile (or whatever other Dockerfile that has the latest `conda` installed)

```Dockerfile
FROM --platform=linux/amd64 debian:stable-slim

RUN apt update -y && apt upgrade -y
RUN apt install curl -y
RUN cd /tmp/ && curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
RUN bash /tmp/Miniconda3-latest-Linux-x86_64.sh -p /opt/conda -b
RUN /opt/conda/bin/conda init --all
RUN /opt/conda/bin/conda install -y -c defaults python conda-build
```

Build and run this docker image...

Once on the command line:

```bash
$ python --version  # let's see which python version we have
Python 3.7.13
$ echo "name: base
channels:
  - defaults
dependencies:
  - python=3.10" > /tmp/environment.yml  # create an environment.yml file with a newer version of Python
$ conda env update -f /tmp/environment.yml
💥  # same error pops up
```

The conda team will be working to addressing this error as soon as possible and will include a fix in the upcoming. We apologize for the turbulence and thank you for flying `conda` 😄 ✈️

**Comment by achimgaedke** [comment]
Thanks @travishathaway - you managed what I tried and failed ...

**Comment by achimgaedke** [comment]
@wolfv - Yep, using `conda<4.14` avoids this bug.

**Comment by wolfv** [comment]
I think we can close this here as it is a `conda` bug and they are working on it.
